### DISCUSSION TITLE
How to load multiple views

### DISCUSSION THREAD

**Discussion by EMwavelength** [mitsuba-renderer/mitsuba3#223] [Q&A]
Hi,
    I like to know how to load multiple images of different viewpoints so that i can render and optimize all reflectance at once instead of optimizing a single view each time, i want to do that automatically so the final reflectance that is saved is not just view dependent, for example a texture optimized from single view only contains reflectance for that view, i want to recover view-independent reflectance from multiple views. Please help me with this as i am still learning.

**Comment by Speierers** [comment]
Hi @EMwavelength ,

I would recommend you take a close look at [this tutorial](https://mitsuba.readthedocs.io/en/latest/src/inverse_rendering/volume_optimization.html) where a volume grid is optimized from different viewpoints simultaneously.
